Teccon1998/GenAIAssistant
# Created Resume Data Extraction

Set up resume data extraction. Used a multi-agent template that allows for easy addition for more functions in process_resume if more functions are desired. To make changes to the way the data is formatted and inserted, edit the prompt in the extract_data() function. To create new agents, and tools refer to [this](https://cookbook.openai.com/examples/structured_outputs_multi_agent) template for context on the existing functions in process_resume.

To test:
Login
Click Generate Insights
Check MongoDB database, collection: files_uploaded

Notes: Used a multi-agent process, but this could be reduced to a single agent process as it currently stands. I think it'd be good though to incorporate it now just in case.
